tools/misc/set_partial_hessian

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
```

Comment:
Just a very quick remark on this script: it uses bash, awk, sed, manually coded option handling and help generation... I think, we should strongly consider to write it in python (and maybe then even make it part of dptools).